packages/opencensus-core/src/exporters/buffer.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as uuidv4 from 'uuid/v4';
+import * as types from './types';
+import * as modelTypes from '../trace/model/types';
+import * as configTypes from '../trace/config/types';
+import * as loggerTypes from '../common/types';
```

Comment:
TODO

Reply:
I understand your point of view. But, the idea here is to standardize the use of "types" module namespace inside core package. If you don't mind, we would like to keep this way, otherwise, it will be a TODO.